config_parser_test.cc

```diff
@@ -9,3 +9,84 @@ TEST(NginxConfigParserTest, SimpleConfig) {
 
   EXPECT_TRUE(success);
 }
+
+// TESTING ToString function of NginxConfigStatement
```

Comment:
May not be necessary tests to evaluate parser functionality but makes sense as additional tests of the class to increase coverage.